tests/benchmark-runner-tests.mjs

```diff
@@ -83,7 +83,9 @@ describe("BenchmarkRunner", () => {
         });
 
         describe("_appendFrame", () => {
-            it("should create an absolutely positioned iframe with 800px x 600px dimensions", async () => {
+            const DEFAULT_WIDTH = 1366;
```

Comment:
I like the refactor a lot!!
Could we re-use the params class and add a failsafe for searchParams if needed? 
Alternatively we could export the hardcoded values from that file and have it all live in the same place? 🤷

Reply:
DONE: Added a `defaultParams` export on the params.mjs module.